docs/operator-manual/server-commands/argocd-application-controller.md

```diff
@@ -55,6 +55,7 @@ argocd-application-controller [flags]
       --redis-insecure-skip-tls-verify         Skip Redis server certificate validation.
       --redis-use-tls                          Use TLS when connecting to Redis. 
       --redisdb int                            Redis database.
+      --repo-error-grace int                   Grace period in seconds for ignoring consecutive errors while communicating with repo server. (default 180)
```

Comment:
The default time is `0` seconds and not `180`.

Reply:
This is autogen and the default is 3mins, same as reconciliation timeout.

Reply:
My bad. missed checking that.